.changeset/late-peas-fail.md

```diff
@@ -0,0 +1,5 @@
+---
+"ponder": patch
+---
+
+Added support for block tags (`startBlock: "latest"`) in `ponder.config.ts`.
```

Comment:
Added support for passing `"latest"` to `startBlock` and `endBlock` in `ponder.config.ts`. With this option, Ponder will fetch the latest block during the build step and use that block number.